aioamqp/exceptions.py

```diff
@@ -49,3 +49,13 @@ def __init__(self, delivery_tag):
     def __repr__(self):
         return 'Publish failed because a nack was received for delivery_tag {}'.format(
             self.delivery_tag)
+
+
+class PublishReturned(AioamqpException):
+    def __init__(self, reply_code, reply_text, exchange, routing_key):
```

Comment:
I could raise PublishFailed, but the repr would be unhelpful and it would be impossible to distinguish a nack from a return.

What would you suggest?